### PR TITLE
Vmos 6847

### PR DESCRIPTION
**Pull request by VladislavDandara25**
Create relationships between Entities

### CODE REVIEW

File: apps/backend/ormconfig.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 module.exports = {
   type: 'mysql',
   host: 'localhost',
-  port: 3307, //port of docker or port of MySQL workbench?
+  port: 3307, //port of MySQL workbench
   username: 'root',
   password: 'root',
   database: 'vitaSQL',
-  entities: ['entity/*.js'],
+  entities: ['data/models/*.ts'],
```
Comment (alexvitamojo) [inline apps/backend/ormconfig.js:8]: write from the root of the project apps/backend...

File: apps/backend/src/data/models/create-modifier-category-entity.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  @Column({ type: 'int' })
+  minSelection: number;
+
+  @Column({ type: 'int' })
+  maxSelection: number;
+
+  @Column({ type: 'int' })
+  productId: number;
+
+  @ManyToOne(() => Product, (product: Product) => product.modifier_categories)
```
Comment (alexvitamojo) [inline apps/backend/src/data/models/create-modifier-category-entity.ts:29]: you are supposed to keep the camelCase here as well

File: apps/backend/src/data/models/create-modifier-category-entity.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  name: string;
+
+  @Column({ type: 'int' })
+  minSelection: number;
+
+  @Column({ type: 'int' })
+  maxSelection: number;
+
+  @Column({ type: 'int' })
+  productId: number;
```
Comment (alexvitamojo) [inline apps/backend/src/data/models/create-modifier-category-entity.ts:27]: you need to map to the correct field in the database. In your database you have `product_id` 
node_modules/typeorm/decorator/options/ColumnOptions.d.ts

**Review by alexvitamojo — COMMENTED** [review]
Model/Entity file name should be identic to the database table but in camelCase 
menu.ts
modifierCategory.ts
Exactly the same goes for repository
menu.ts
modifierCategory.ts
but the repository class must have Repository suffix `MenuRepository`

File: apps/backend/src/data/models/create-modifier-category-entity.ts (hunk 3 above)
Comment (VladislavDandara25) [reply apps/backend/src/data/models/create-modifier-category-entity.ts]: Fixed

**Pull request merged**